sno/working_copy.py

```diff
@@ -330,6 +331,30 @@ def read_meta(self, dataset):
         with self.session() as db:
             yield from gpkg.get_meta_info(db, dataset.name)
 
+    def iter_meta_items_v2(self, dataset):
+        from .gpkg_adapter import wkt_to_srs_str, gpkg_to_v2_schema
+
+        gpkg_meta_items = dict(self.read_meta(dataset))
+        identifier = gpkg_meta_items["gpkg_contents"]["identifier"]
+        # FIXME: a better way of roundtripping identifiers?
```

Comment:
any idea why we add table-name prefixes to identifiers? I found that surprising. Can we strip it?

Reply:
This could use improvement (hence the FIXME), but won't do it in this PR. These identifiers need to be unique in GPKG and using the user-supplied title as the identifier doesn't guarantee uniqueness. Not sure what the right solution is.